### PR TITLE
refactor: first batch of landing page improvements

### LINKED ISSUE #474: Fix lighthouse report
Currently KodaDot has not so good lighthouse stats.

Screenshot 2021-06-18 at 11 57 45

Deliverables:
- [ ] Fix performance
- [ ] Optional: fix other fields

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #474
- [x] first batch of improvements. this will require lots if increments

### Before submitting Pull Request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by yangwao** [comment]
What improvements inside 😜

**Comment by kkukelka** [comment]
> What improvements inside 😜

no legacy js, lazy imports of landing page components, modern image formats & fixed sizes, few seo & a11y stuff

**Comment by kkukelka** [comment]
@yangwao wrote a list of all iterations that will have to be made to improve pagespeed https://github.com/kodadot/nft-gallery/issues/2816

**Comment by yangwao** [comment]
I've tested this meanwhile and found improvements! 

Let's merge this tho! :)

File: components/landing/Landing.vue [hunk 1]
```diff
@@ -4,10 +4,11 @@
       <div class="columns">
         <div class="column">
           <img
-            src="~/assets/KODADOT.png"
+            src="~/assets/KODADOT.webp"
```
Comment (prachi00) [inline components/landing/Landing.vue:7]: https://caniuse.com/webp

also, I see on older versions and on safari (for pre big sur versions) there is partial support for webp, would that be an issue then?

Comment (kkukelka) [reply components/landing/Landing.vue]: you are absolutely right. Just added `.png` fallback for Safari pre-"Big Sur"
also fixed merge conflicts

**Review by prachi00 — APPROVED** [review]

**Comment by prachi00** [comment]
pay 200 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $200 @ 155.81 USD/KSM ~ 1.284 $KSM
🧗 [EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW](https://kusama.subscan.io/account/EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)
🔗 [0x4f6ee9b467d40c2681471a6681245984c30789de8a3ef13b7aeaddde1a48884b](https://kusama.subscan.io/extrinsic/0x4f6ee9b467d40c2681471a6681245984c30789de8a3ef13b7aeaddde1a48884b)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**